com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -56,10 +55,12 @@ internal void GenerateGlobalObjectIdHash()
         /// </summary>
         internal NetworkManager NetworkManagerOwner;
 
+        private ulong m_NetworkObjectId;
+
         /// <summary>
         /// Gets the unique Id of this object that is synced across the network
         /// </summary>
-        public ulong NetworkObjectId { get; internal set; }
+        public ulong NetworkObjectId { get => m_NetworkObjectId; internal set => m_NetworkObjectId = value; }
```

Comment:
why? I don't see any difference between extracting this into a local member vs leaving it as it is?

Reply:
This was part of the failed attempt to implement INetworkSerializable on NetworkObject because the serializer can't take a property by ref. I'm going to revert it.

Reply:
Reverted.